src/components/Text.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Heading, HeadingDisabled } from './Text'
+
+storiesOf('Text', module).add('Heading', () => (
+    <Heading>Heading Text</Heading>
+))
+
+storiesOf('Text', module).add('Heading disabled', () => (
+    <HeadingDisabled>Heading Text</HeadingDisabled>
```

Comment:
I would combine the Heading and the HeadingDisabled component into one and pass the disabled state as a prop to it:

```suggestion
    <Heading disable={true}>Heading Text</Heading>
    
    // short version ={true} can be omitted
    <Heading disable>Heading Text</Heading>
```